lib/ace/ext/papeeria/latex_parsing_context.coffee

```diff
@@ -9,24 +9,26 @@ define((require, exports, module) ->
      *
      * Returns context at row.
     ###
-    getContext = (session, row) ->
+    # Specific for token"s system of type in ace
+  # We saw such a realization in html_completions.js
+    isType = (token, type) ->
+        return token.type.split(".").indexOf(type) > -1
+
+    getContext = (session, row, column) ->
+        token = session.getTokenAt(row, column)
+        if token? and isType(token, "math")
+            return EQUATION_STATE
+        else
+            getContext_(session, row)
+
+    getContext_ = (session, row) ->
```

Comment:
`getContextFromPos` and `getContextFromRow` must be better names than `getContext` and `getContext_`